### ISSUE TITLE
Possible user confusion with EEPROM.BIN

### ISSUE THREAD

**Issue by projectkk2glider** [opentx/opentx#1282]
I didn't spot this before (I am on Linux), but in testing issue #1281 I spotted a potential flaw.

The .BIN files seem to be associated with Companion after its installation. This leads to one serious and easy mistake. 

When user connects Taranis in bootloader mode, he is presented with a drive which contains EEPROM.BIN and FIRMWARE.BIN. He could easily double click on any one of those files, Companion would start and open that file. If it happens to be EEPROM.BIN it will politely display model settings. User thinks he is doing alright. 

But writing this back to transmitter would mean doing read and write to/from the same file. This could also result in `access denied` problem encountered in #1281.

Bottom line is: Companion should refuse to open EEPROM.BIN file if it is located on Taranis.
